.github/workflows/ci.yml

```diff
@@ -45,27 +45,27 @@ jobs:
             omp: n
             mpi: n
           - dagmc: y
-            python-version: '3.10'
+            python-version: "3.10"
```

Comment:
Out of curiosity, why the change from single to double quotes?

Reply:
My editor deciding for me that double quotes are better 🤷‍♂️

Reply:
Right on! Just making sure there wasn't a technical difference.